Formula/yh.rb

```diff
@@ -0,0 +1,24 @@
+class Yh < Formula
+  desc "YAML syntax highlighter to bring colours where only jq could"
+  homepage "https://github.com/andreazorzetto/yh"
+  url "https://github.com/andreazorzetto/yh/archive/v0.4.0.tar.gz"
+  sha256 "78ef799c500c00164ea05aacafc5c34dccc565e364285f05636c920c2c356d73"
+  license "Apache-2.0"
+
+  head "https://github.com/andreazorzetto/yh.git"
+
+  livecheck do
+    url "https://github.com/andreazorzetto/yh/releases/latest"
+    regex(%r{href=.*?/tag/v((0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*))?}i)
+  end
+
```

Comment:
we can remove it for now
```suggestion
```

Reply:
I will remove it 👍 Could you please tell me why it is not needed at least now?

Reply:
the default livecheck is sufficient enough.

Reply:
default livecheck output
```
URL:              https://github.com/andreazorzetto/yh.git
Strategies:       Git
Strategy:         Git

Matched Versions:
v0.1 => #<Version:0x00007f9866909e70 @version="0.1", @detected_from_url=false>
v0.1.1 => #<Version:0x00007f9866909d80 @version="0.1.1", @detected_from_url=false>
v0.2.0 => #<Version:0x00007f9866909c68 @version="0.2.0", @detected_from_url=false>
v0.2.1 => #<Version:0x00007f9866909b50 @version="0.2.1", @detected_from_url=false>
v0.3.1 => #<Version:0x00007f9866909a60 @version="0.3.1", @detected_from_url=false>
v0.3.2 => #<Version:0x00007f9866909970 @version="0.3.2", @detected_from_url=false>
v0.3.3 => #<Version:0x00007f9866909880 @version="0.3.3", @detected_from_url=false>
v0.4.0 => #<Version:0x00007f9866909790 @version="0.4.0", @detected_from_url=false>
yh (guessed) : 0.4.0 ==> 0.4.0
```